resources/windows/wix/directorylist.wxs.erb

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+  <Fragment>
+
+  <%- generate_wix_dirs(project) -%>
```

Comment:
Three things:
- You're going to want to start this with `<%=`
- to access the wix_dirs method, you need to call it like `@platform.generate_wix_dirs`
- from what I can tell, we don't actually have access to the project object at this point. We do, however, have access to `get_directories`

```
[4] pry(#<Vanagon::Project>)> get_directories
=> [#<Vanagon::Windows::Pathname:0x007fa596992a30 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/ProgramFiles64Folder/PuppetLabs", @wix_id=nil>,
 #<Vanagon::Windows::Pathname:0x007fa596992210 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/ProgramFiles64Folder/PuppetLabs/Puppet", @wix_id=nil>,
 #<Vanagon::Windows::Pathname:0x007fa596991928 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs", @wix_id=nil>,
 #<Vanagon::Windows::Pathname:0x007fa5969910e0 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/ProgramFiles64Folder/PuppetLabs/bin", @wix_id=nil>,
 #<Vanagon::Common::Pathname:0x007fa596ba3798 @config=nil, @group=nil, @mode="0750", @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/puppet/cache">,
 #<Vanagon::Common::Pathname:0x007fa596ba2e38 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/puppet/etc">,
 #<Vanagon::Common::Pathname:0x007fa596ba25c8 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/code">,
 #<Vanagon::Common::Pathname:0x007fa596ba1da8 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/code/modules">,
 #<Vanagon::Common::Pathname:0x007fa596ba14c0 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/ProgramFiles64Folder/PuppetLabs/Puppet/modules">,
 #<Vanagon::Common::Pathname:0x007fa596ba08e0 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/code/environments">,
 #<Vanagon::Common::Pathname:0x007fa596b979c0 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/code/environments/production">,
 #<Vanagon::Common::Pathname:0x007fa596b96ac0 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/code/environments/production/manifests">,
 #<Vanagon::Common::Pathname:0x007fa596b95f08 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/code/environments/production/modules">,
 #<Vanagon::Common::Pathname:0x007fa596b95378 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/puppet/var/log">,
 #<Vanagon::Common::Pathname:0x007fa596b94860 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/puppet/var/run">,
 #<Vanagon::Common::Pathname:0x007fa596b3dad8 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/facter/facts.d">,
 #<Vanagon::Common::Pathname:0x007fa596b32cc8 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/code/hieradata">,
 #<Vanagon::Common::Pathname:0x007fa596aefc70 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/mcollective/etc">,
 #<Vanagon::Common::Pathname:0x007fa596aef270 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/mcollective/plugins">,
 #<Vanagon::Common::Pathname:0x007fa596aee668 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/mcollective/var/log">,
 #<Vanagon::Common::Pathname:0x007fa596a6c078 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/pxp-agent">,
 #<Vanagon::Common::Pathname:0x007fa596a631d0 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/pxp-agent/etc/modules">,
 #<Vanagon::Common::Pathname:0x007fa596a62168 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/pxp-agent/var/spool">,
 #<Vanagon::Common::Pathname:0x007fa596a612b8 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/pxp-agent/var/log">,
 #<Vanagon::Common::Pathname:0x007fa596a60278 @config=nil, @group=nil, @mode=nil, @owner=nil, @path="C:/CommonAppDataFolder/PuppetLabs/pxp-agent/var/run">]
```
Rather than passing in project, could we pass in an array of directories to parse?